minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -259,6 +259,7 @@ private void startMaster(TachyonConf testConf) throws IOException {
     if (ufs.getClass().getSimpleName().equals("LocalMiniDFSCluster")) {
       testConf.set(Constants.UNDERFS_ADDRESS, ufs.getUnderFilesystemAddress() + mTachyonHome);
       MasterContext.reset(testConf);
+      mMasterConf = testConf;
```

Comment:
You're right, we were losing that configuration - fixed.

I'm looking forward to the day when all test configuration can be done in one class.